src/netconfig.cpp

```diff
@@ -322,6 +322,7 @@ static void cmdVlan(Dbus& bus, Arguments& args)
     puts(completeMessage);
 }
 
+/** @brief Configure remote syslog server: `set ADDR[:PORT]` */
```

Comment:
This comment and the such below also don't seem to me relevant for the commit message.

Reply:
Done